docs/installation.rst

```diff
@@ -198,10 +174,10 @@ To start the dev server, run ``./manage.py runserver``, then open up
 ``http://localhost:8000``. If everything's working, you should see
 the MDN home page!
 
-You might need to first set ``LC_CTYPE`` if you're on Mac OS X until
-`bug 754728 <https://bugzilla.mozilla.org/show_bug.cgi?id=754728>`_ is fixed::
+You might need to first set ``LC_CTYPE`` in your ``.env`` file if you're on
+Mac OS X until `bug 754728 <https://bugzil.la/754728>`_ is fixed::
```

Comment:
I don't know since the ticket doesn't really explain why this is needed in the first place. The commit referenced in the ticket (5f745996c009f658fda9ab359d6e0d7c4e56fd2e) doesn't really explain it either. I don't see a reason to not keep that info.